bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/page/chat-message/component.tsx

```diff
@@ -110,6 +110,10 @@ const ChatMesssage: React.FC<ChatMessageProps> = ({
     || lastSenderPreviousPage) === message?.user?.userId;
   const isSystemSender = message.messageType === ChatMessageType.BREAKOUT_ROOM;
   const dateTime = new Date(message?.createdAt);
+
+  const msgTime = dateTime.toLocaleTimeString('en-US', { hour: 'numeric', minute: 'numeric', hour12: true });
+  const clearMessage = `${intl.formatMessage(intlMessages.chatClear)} at ${msgTime}`;
```

Comment:
![2024-01-26_14-18](https://github.com/bigbluebutton/bigbluebutton/assets/3728706/8e17bca6-5dbe-432f-9de4-a5f4fda5191a)

the "at" part is not going to be translated to other languages